lib/feature/pokemon_details_page.dart/widgets/evolution_tab/widgets/pokemon_evolution_tab_widget.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex_async_redux/api/models/species_model.dart';
+import 'package:pokedex_async_redux/utilities/constants.dart';
+import 'package:pokedex_async_redux/utilities/extensions.dart';
+
+class PokemonEvolutionWidget extends StatelessWidget {
+  const PokemonEvolutionWidget({
+    required this.thisSpecies,
+    required this.arrowCheck,
+  });
+  final List<Species> thisSpecies;
+  final bool arrowCheck;
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: thisSpecies
+          .map(
+            (thisSpecies) => Container(
+              margin: arrowCheck ? EdgeInsets.fromLTRB(30, 0, 0, 0) : EdgeInsets.all(0),
+              height: 80,
+              width: 80,
+              child: Stack(
+                clipBehavior: Clip.none,
+                children: [
+                  if (arrowCheck)
+                    Positioned(
+                      left: -55,
+                      child: Container(
```

Comment:
remove the container. Assign the bottom and top of the positioned widget to 0, so it centres vertically